### ISSUE TITLE
how to download a password-protected view-only video?

### ISSUE THREAD

**Issue by anthonywonglc** [kylon/Sharedown#98]
I have a sharepoint video link which requires a password to open. After installing sharedown and adding the link to it, I clicked start download.

A chrome browser pop up. I input the password, sharedown keeps loading but the video is not downloaded.

Anyone can help? Thanks.

**Comment by anthonywonglc** [comment]
No. It requires a password in order to access to a folder that contains the video that I want to download.

**Comment by kylon** [comment]
i'm back, does it load the video page? it will wait for the video player, so i don't see why it should not work in this case
can you post a log?
